src/components/ux-joyride/ux-joyride.hbs

```diff
@@ -1,9 +1,10 @@
 <div class="ux-joyride">
 	<a on-tap-enter="toggle"
+		on-mouseover-mouseout="{{#if showOnHover}}hover{{/if}}"
```

Comment:
Would it be better to write that as

```html
{{#if showOnHover}}on-mouseover-mouseout="hover"{{/if}}
```

As the event would only be listened to if desired, rather than as written the event always being listened to and only when desired would the handler be set.

Reply:
The event listener didn't get initiated if we wrapped it in the condition statement, that's why we need to stick to the original version.